app/src/main/java/com/xa/xpensauditor/TabFragment.java

```diff
@@ -58,7 +66,13 @@ public TabFragment() {
         // Required empty public constructor
     }
 
-
+    /**
+     * Loads data when fragment is loaded
+     * @param inflater
+     * @param container
+     * @param savedInstanceState
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.NonEmptyAtclauseDescriptionCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>At-clause should have a non-empty description.